src/main/java/net/datafaker/providers/base/Locality.java

```diff
@@ -2,39 +2,32 @@
 
 import net.datafaker.annotations.Deterministic;
 
-import java.io.File;
-import java.io.IOException;
-import java.lang.management.ManagementFactory;
-import java.nio.file.FileSystem;
-import java.nio.file.FileSystems;
-import java.nio.file.FileVisitResult;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.nio.file.SimpleFileVisitor;
-import java.nio.file.attribute.BasicFileAttributes;
 import java.util.ArrayList;
 import java.util.Collections;
-import java.util.HashSet;
 import java.util.List;
 import java.util.Locale;
 import java.util.Random;
 import java.util.Set;
-import java.util.function.Predicate;
-import java.util.logging.Logger;
-
-import static java.util.Collections.emptySet;
 
 /**
  * Generates random locales in different forms.
  *
  * @since 1.7.0
  */
 public class Locality extends AbstractProvider<BaseProviders> {
-    private static final Logger log = Logger.getLogger(Locality.class.getName());
-    private static final Predicate<String> SCAN_ALL_JARS = name -> true;
+    private static final List<String> locales = List.of(
```

Comment:
Should be "LOCALES"

Reply:
Was the list generated programmatically? (If so it might be worth commenting about that for future maint)

Reply:
1. Renamed to `LOCALES`.
2. I added a hint to error message, it should help.

For example, if I add a file `src/main/resources/_FOO.yml`, but forget to add `_foo` to constant `Locality.LOCALES`, then one test will fail with a very clear instruction:

```java
java.lang.AssertionError: [Apparently, somebody forgot to add the new locale to Locality.LOCALES] 
Expecting actual:
  ["_al", "_bg",    .....]
to contain exactly in any order:
  ["ro-md", ... ]
but could not find the following elements:
  ["_foo"]
```

Reply:
Great, okay!